.github/workflows/k3d-ci.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
             -p "30870:30870@agent:0:direct"
             --agents 1
             --no-lb
-            --k3s-arg "--disable=traefik,servicelb@server:*"
+            --k3s-arg "--disable=traefik,servicelb,metrics-server@server:*"
```

Comment:
Still a little unclear what we're doing with this, do we want it enabled in CI?

Reply:
I think we do only if we're testing something in particular, currently it doesn't affect existing tests, and more likely to use up mem on the GH runner, so thought should disable it for now.

Maybe we enable in nightly tests and specific tests once we have autoscaling.